taigit/src/libraries/Taiga.ts

```diff
@@ -78,4 +78,142 @@ task_history(taskId : number) : Promise<Object> {
     }
 
     return output;
+}
+
+/**
+ * @summary Get the Task count with its status for each sprint
+ * @param projName project/slug Name
+ * @returns array of each members along with task count for all sprints 
+ * {
+ *      inProgressTaskCount: number
+ *      name: string
+ *      newTaskCount: number
+ *      sprintName: string
+ *      taskReadyForTestCount: number
+ *      taskWithUnknownStatusCount: number
+ *      totalTaskCount: number 
+ * }
+ */
+export async function
+getTaskStatusCount(projName: string): Promise<Object> {
+    let response = await axios.get('https://api.taiga.io/api/v1/projects/by_slug?slug=' + projName);
```

Comment:
I'm wondering if it is necessary to have to keep having slug and id API calls when they could be passed in from the parameters...

Reply:
We know that front end team has slug name, but I was not sure whether while calling this API they will have the projectId with them.

So, do you suggest to use only projectId instead of slug name?

Reply:
I do. But I will also cross check with frontend. Will post the question on group slack so you can contribute to the conversation